sql/ttl/ttljob: TestRowLevelTTLJobMultipleNodes failed

sql/ttl/ttljob.TestRowLevelTTLJobMultipleNodes [failed](https://mesolite.cluster.engflow.com/invocations/default/c7736c78-fc0a-4554-b461-33de18952e1d?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc3FsL3R0bC90dGxqb2I6dHRsam9iX3Rlc3Q=) on master @ [377adba1a7814b2b18b3ad19462efbd9c8226b98](https://github.com/cockroachdb/cockroach/commits/377adba1a7814b2b18b3ad19462efbd9c8226b98):


```
=== RUN   TestRowLevelTTLJobMultipleNodes
    test_log_scope.go:172: test logs captured to: outputs.zip/logTestRowLevelTTLJobMultipleNodes987672716
    test_log_scope.go:82: use -show-logs to present logs inline
    ttljob_test.go:728: -- test log scope end --
test logs left over in: outputs.zip/logTestRowLevelTTLJobMultipleNodes987672716
--- FAIL: TestRowLevelTTLJobMultipleNodes (58.01s)
=== RUN   TestRowLevelTTLJobMultipleNodes/2_splits_(min_cluster_version)
    ttljob_test.go:204: 
        	Error Trace:	pkg/sql/ttl/ttljob/ttljob_test.go:204
        	            				pkg/sql/ttl/ttljob/ttljob_test.go:720
        	Error:      	Should be true
        	Test:       	TestRowLevelTTLJobMultipleNodes/2_splits_(min_cluster_version)
        	Messages:   	
        	            	expectedStatus="succeeded"
        	            	actualStatuses="failed"
        	            	 expectedError=""
        	            	  actualErrors="incorrect number of span partitions expected=3 actual=1
        	            	HINT: You have encountered an unexpected error.
        	            	
        	            	Please check the public issue tracker to check whether this problem is
        	            	already tracked. If you cannot find it there, please report the error
        	            	with details by creating a new issue.
        	            	
        	            	If you would rather not post publicly, please contact us directly
        	            	using the support form.
        	            	
        	            	We appreciate your feedback.
        	            	"
--- FAIL: TestRowLevelTTLJobMultipleNodes/2_splits_(min_cluster_version) (6.62s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRowLevelTTLJobMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-58804

sql/ttl/ttljob.TestRowLevelTTLJobMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/21007899?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/21007899?buildTab=artifacts#/) on master @ [c6ec5849dfea61702bc94c5b794a7659c0e914f4](https://github.com/cockroachdb/cockroach/commits/c6ec5849dfea61702bc94c5b794a7659c0e914f4):


```
=== RUN   TestRowLevelTTLJobMultipleNodes
    test_log_scope.go:172: test logs captured to: /artifacts/tmp/_tmp/772d54cb8543909a2bf6361ebbdd57e0/logTestRowLevelTTLJobMultipleNodes3967963711
    test_log_scope.go:82: use -show-logs to present logs inline
    ttljob_test.go:728: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/772d54cb8543909a2bf6361ebbdd57e0/logTestRowLevelTTLJobMultipleNodes3967963711
--- FAIL: TestRowLevelTTLJobMultipleNodes (22.51s)
=== RUN   TestRowLevelTTLJobMultipleNodes/2_splits_(min_cluster_version)
    ttljob_test.go:204: 
        	Error Trace:	pkg/sql/ttl/ttljob/ttljob_test.go:204
        	            				pkg/sql/ttl/ttljob/ttljob_test.go:720
        	Error:      	Should be true
        	Test:       	TestRowLevelTTLJobMultipleNodes/2_splits_(min_cluster_version)
        	Messages:   	
        	            	expectedStatus="succeeded"
        	            	actualStatuses="failed"
        	            	 expectedError=""
        	            	  actualErrors="incorrect number of span partitions expected=3 actual=1
        	            	HINT: You have encountered an unexpected error.
        	            	
        	            	Please check the public issue tracker to check whether this problem is
        	            	already tracked. If you cannot find it there, please report the error
        	            	with details by creating a new issue.
        	            	
        	            	If you would rather not post publicly, please contact us directly
        	            	using the support form.
        	            	
        	            	We appreciate your feedback.
        	            	"
--- FAIL: TestRowLevelTTLJobMultipleNodes/2_splits_(min_cluster_version) (3.48s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRowLevelTTLJobMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/ttl/ttljob.TestRowLevelTTLJobMultipleNodes [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/21009274?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/21009274?buildTab=artifacts#/) on master @ [ac94a93a76e9df81c3d9bd3990d42d6d6fefef3d](https://github.com/cockroachdb/cockroach/commits/ac94a93a76e9df81c3d9bd3990d42d6d6fefef3d):


```
=== RUN   TestRowLevelTTLJobMultipleNodes
    test_log_scope.go:172: test logs captured to: /artifacts/tmp/_tmp/772d54cb8543909a2bf6361ebbdd57e0/logTestRowLevelTTLJobMultipleNodes3681801045
    test_log_scope.go:82: use -show-logs to present logs inline
    ttljob_test.go:728: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/772d54cb8543909a2bf6361ebbdd57e0/logTestRowLevelTTLJobMultipleNodes3681801045
--- FAIL: TestRowLevelTTLJobMultipleNodes (23.01s)
=== RUN   TestRowLevelTTLJobMultipleNodes/2_splits
    test_server_shim.go:183: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    ttljob_test.go:204: 
        	Error Trace:	pkg/sql/ttl/ttljob/ttljob_test.go:204
        	            				pkg/sql/ttl/ttljob/ttljob_test.go:720
        	Error:      	Should be true
        	Test:       	TestRowLevelTTLJobMultipleNodes/2_splits
        	Messages:   	
        	            	expectedStatus="succeeded"
        	            	actualStatuses="failed"
        	            	 expectedError=""
        	            	  actualErrors="incorrect number of span partitions expected=3 actual=1
        	            	HINT: You have encountered an unexpected error.
        	            	
        	            	Please check the public issue tracker to check whether this problem is
        	            	already tracked. If you cannot find it there, please report the error
        	            	with details by creating a new issue.
        	            	
        	            	If you would rather not post publicly, please contact us directly
        	            	using the support form.
        	            	
        	            	We appreciate your feedback.
        	            	"
--- FAIL: TestRowLevelTTLJobMultipleNodes/2_splits (4.28s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRowLevelTTLJobMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

sql/ttl/ttljob.TestRowLevelTTLJobMultipleNodes [failed](https://mesolite.cluster.engflow.com/invocations/default/c25a2306-db1b-4ed4-b226-a4e0b9227bd7?testReportRun=3&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc3FsL3R0bC90dGxqb2I6dHRsam9iX3Rlc3Q=) on master @ [ac94a93a76e9df81c3d9bd3990d42d6d6fefef3d](https://github.com/cockroachdb/cockroach/commits/ac94a93a76e9df81c3d9bd3990d42d6d6fefef3d):


```
=== RUN   TestRowLevelTTLJobMultipleNodes
    test_log_scope.go:172: test logs captured to: outputs.zip/logTestRowLevelTTLJobMultipleNodes3999424761
    test_log_scope.go:82: use -show-logs to present logs inline
    ttljob_test.go:728: -- test log scope end --
test logs left over in: outputs.zip/logTestRowLevelTTLJobMultipleNodes3999424761
--- FAIL: TestRowLevelTTLJobMultipleNodes (68.67s)
=== RUN   TestRowLevelTTLJobMultipleNodes/2_splits
    test_server_shim.go:183: automatically injected a shared process virtual cluster under test; see comment at top of test_server_shim.go for details.
    ttljob_test.go:204: 
        	Error Trace:	pkg/sql/ttl/ttljob/ttljob_test.go:204
        	            				pkg/sql/ttl/ttljob/ttljob_test.go:720
        	Error:      	Should be true
        	Test:       	TestRowLevelTTLJobMultipleNodes/2_splits
        	Messages:   	
        	            	expectedStatus="succeeded"
        	            	actualStatuses="failed"
        	            	 expectedError=""
        	            	  actualErrors="incorrect number of span partitions expected=3 actual=1
        	            	HINT: You have encountered an unexpected error.
        	            	
        	            	Please check the public issue tracker to check whether this problem is
        	            	already tracked. If you cannot find it there, please report the error
        	            	with details by creating a new issue.
        	            	
        	            	If you would rather not post publicly, please contact us directly
        	            	using the support form.
        	            	
        	            	We appreciate your feedback.
        	            	"
--- FAIL: TestRowLevelTTLJobMultipleNodes/2_splits (11.21s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=3</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRowLevelTTLJobMultipleNodes.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/sql/ttl/ttljob/ttljob_test.go
+++ b/pkg/sql/ttl/ttljob/ttljob_test.go
@@ -87,7 +87,10 @@ func newRowLevelTTLTestJobTestHelper(
 		return st
 	}
 
-	jobsInterval := 2 * time.Second
+	jobsInterval := 5 * time.Second
+	if skip.Duress() {
+		jobsInterval = 30 * time.Second
+	}
 	requestFilter, _ := testutils.TestingRequestFilterRetryTxnWithPrefix(t, "ttljob-", 1)
 	baseTestingKnobs := base.TestingKnobs{
 		Server: &server.TestingKnobs{

